src/delay.h

```diff
@@ -28,4 +28,9 @@
 int uperf_sleep(hrtime_t);
 int uperf_spin(hrtime_t);
 
+/*
+ * Initialize alarm to be able to wake up the device between transactions / flowops
+ */
+int init_android_alarm();
```

Comment:
For now yes, will put it under ifdef. Maybe in future we'll want to use alarms for sleeping on Linux as well. Then device under test will be able to suspend during think operations.